### PR TITLE
Updating changelog

### PR DESCRIPTION
**Pull request by aaronjeline**
## Description of changes
Adds 2.4.3 info to changelog
## Issue #, if available
N/A
## Checklist for requesting a review

The change in this PR is (choose one, and delete the other options):

- [X] A change "invisible" to users (e.g., documentation, changes to "internal" crates like `cedar-policy-core`, `cedar-validator`, etc.)

I confirm that this PR (choose one, and delete the other options):

- [X] Updates the "Unreleased" section of the CHANGELOG with a description of my change (required for major/minor version bumps).

I confirm that [`cedar-spec`](https://github.com/cedar-policy/cedar-spec) (choose one, and delete the other options):

- [X] Does not require updates because my change does not impact the Cedar Dafny model or DRT infrastructure.

## Disclaimer

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cedar-policy/CHANGELOG.md [hunk 1]
```diff
@@ -154,6 +154,15 @@ Cedar Language Version: 3.0.0
...
   feature flag. (#428)
 
+  
+## [2.4.3] - 2023-12-21
+
+Cedar Language Version: 2.1.3
+
+### Fixed
+
+- Reverted accidental breaking change to schema format introduced in the 2.3.2
```
Comment (cdisselkoen) [inline cedar-policy/CHANGELOG.md:164]: looks like the sentence got cut off

File: cedar-policy/CHANGELOG.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ Cedar Language Version: 3.0.0
 - `Entities::from_*()` methods now validate the entities against the `schema`,
   if a `schema` is provided. (#360)
 - `Entities::from_entities()` and `Entities::add_entities()` now take an
-  optional schema argument. (#360)
+ 8 optional schema argument. (#360)
```
Comment (cdisselkoen) [inline cedar-policy/CHANGELOG.md:100]: unintentional I assume

File: cedar-policy/CHANGELOG.md [hunk 3]
```diff
@@ -113,7 +113,7 @@ Cedar Language Version: 3.0.0
   multiple `is_authorized` calls). As a result, it returns `Result`, because
   attribute evaluation can fail. (#430)
 - `Entities::from_json_*()` also now eagerly evaluates entity attributes, and as
-  a result returns errors when attribute evaluation fails. (#430)
+  8a result returns errors when attribute evaluation fails. (#430)
```
Comment (cdisselkoen) [inline cedar-policy/CHANGELOG.md:116]: unintentional I assume

File: cedar-policy/CHANGELOG.md [hunk 4]
```diff
@@ -128,7 +128,7 @@ Cedar Language Version: 3.0.0
   now return `&SourceLocation<'a>` instead of `&'a PolicyID`, matching
   `ValidationError::location`. (#405)
 - `ValidationWarningKind` is now `non_exhaustive`, allowing future warnings to
-  be added without a breaking change. (#404)
+  b8e added without a breaking change. (#404)
```
Comment (cdisselkoen) [inline cedar-policy/CHANGELOG.md:131]: unintentional I assume

**Review by cdisselkoen — APPROVED** [review]

**Review by john-h-kastner-aws — APPROVED** [review]

**Pull request merged**